scripts/release/debian/test_deb_in_docker.sh

```diff
@@ -8,7 +8,8 @@ export USERNAME=azureuser
 apt update
 apt install -y apt-transport-https git gcc python3-dev
 
-dpkg -i /mnt/artifacts/azure-cli_$CLI_VERSION-1~${DISTRO}_all.deb
+DEB_NAME=$(find /mnt/artifacts -type f -name "azure-cli_$CLI_VERSION-1~${DISTRO}_*.deb")
+dpkg -i $DEB_NAME
```

Comment:
I think

```
dpkg -i /mnt/artifacts/azure-cli_$CLI_VERSION-1~${DISTRO}_*.deb
```

may simply do the trick.

Also see:

- https://unix.stackexchange.com/questions/156205/how-can-i-get-the-first-match-from-wildcard-expansion
- https://unix.stackexchange.com/questions/213812/parameter-expansion-in-variable-assigned-with-a-wildcard